go/cmd/dolt/commands/sqlserver/serverconfig.go

```diff
@@ -217,22 +222,29 @@ func (cfg *commandLineServerConfig) PersistenceBehavior() string {
 	return cfg.persistenceBehavior
 }
 
+// TLSKey returns a path to the servers PEM-encoded private TLS key. "" if there is none.
 func (cfg *commandLineServerConfig) TLSKey() string {
 	return cfg.tlsKey
 }
 
+// TLSCert returns a path to the servers PEM-encoded TLS certificate chain. "" if there is none.
 func (cfg *commandLineServerConfig) TLSCert() string {
 	return cfg.tlsCert
 }
 
+// RequireSecureTransport is true if the server should reject non-TLS connections.
 func (cfg *commandLineServerConfig) RequireSecureTransport() bool {
 	return cfg.requireSecureTransport
 }
 
+// DisableClientMultiStatements is true if we want the server to not
```

Comment:
AWESOME godocs! Thank you for helping fill these in while you were working in this file! 🎉

Reply:
Thank you for suggesting this! I think this is really helpful, I'm going to try and make this a habit :)